akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -990,7 +990,28 @@ trait FlowOps[+Out, +Mat] {
    * '''Cancels when''' downstream completes
    */
   def groupedWithin(n: Int, d: FiniteDuration): Repr[immutable.Seq[Out]] =
-    via(new GroupedWithin[Out](n, d))
+    via(new GroupedWeightedWithin[Out](n, ConstantFun.oneLong, d).withAttributes(DefaultAttributes.groupedWithin))
```

Comment:
Since graph stages are reusable could make this a singleton instead of creating a new instance every time it is used, just put it as an `object` next to the `GroupedWeightedWithin` one, then you can also override `initialAttributes` with the default attributes for this.